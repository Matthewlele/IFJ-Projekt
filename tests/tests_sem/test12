intNum = 5
floatNum = 3.14

shouldBeFloat = intNum + floatNum
print shouldBeFloat

print "\n"

if (shouldBeFloat != intNum) == true then x = 2
else
  print "this is false"
end
