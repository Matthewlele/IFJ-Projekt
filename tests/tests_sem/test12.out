expected SYNTAX ERR: line 9
